locales/base/translation.json

```diff
@@ -122,6 +122,7 @@
   "editProfile": "Edit Profile",
   "confirmNumber": "Connect phone number",
   "cancel": "Cancel",
+  "importToken": "Import Token",
```

Comment:
can you please check with large device fonts enabled that the layout is okay? the assets screen has a scroll aware header and i think we may need to rethink this copy

Reply:
| Standard | Larger Text | Larger Accessibility |
| -- | -- | -- |
| ![image](https://github.com/valora-inc/wallet/assets/147835134/47cec31b-bd42-4c5a-abc0-d75bbd8aa3a9) | ![image](https://github.com/valora-inc/wallet/assets/147835134/b507cc1e-ab47-41b5-a943-1e45d69c9985) | ![image](https://github.com/valora-inc/wallet/assets/147835134/7f2d8b66-9dd0-4246-aea2-03c876f423a1) |

Reply:
hmm yeah i guess the definition of what is acceptable layout is subjective, if Nitya is okay with this then fine. but i personally think that we'd be better off with just "Import" so that it doesn't push the (more important) asset balance off center